Jenkinsfile.nightly

```diff
@@ -260,6 +278,18 @@ stage('Send email') {
         subject: '$PROJECT_NAME - Build # $BUILD_NUMBER',
         body: bodytxt
     )
+    bodytxt = """
+        ${projectName} - Build # ${env.BUILD_NUMBER}
+        Check console output at ${env.JOB_URL}${env.BUILD_NUMBER}/ to view the results.
+        Bond apk Uploaded to: http://repository.cliqz.com.s3.amazonaws.com/dist/android/nightly/bond/latest.apk
+        Bond X86 apk Uploaded to: http://repository.cliqz.com.s3.amazonaws.com/dist/android/nightly/bond/latest_x86.apk
+        ${changesSinceLastSuccess}
+        Issues Transtioned ${transitionIssueList}""".stripIndent()
+    emailext(
+        to: 'javier@cliqz.com stefano@cliqz.com ravjit@cliqz.com dominik.s@cliqz.com alina@cliqz.com inci@cliqz.com richard@cliqz.com moaz@cliqz.com oleksandra@cliqz.com',
```

Comment:
Do not hardcode personal addresses in a Jenkins job, we can always create another mailing list